src/lib.rs

```diff
@@ -57,13 +60,15 @@ mod tests {
         async fn get_json<T: DeserializeOwned>(
             &self,
             url: &str,
-            api_key: Option<String>,
+            header: http::header::HeaderMap,
         ) -> Result<T> {
-            let request = isahc::Request::get(Uri::from_str(url)?)
-                .header("AUTHORIZATION", api_key.unwrap_or_default())
-                .body(())
-                .unwrap();
-            let json = isahc::send_async(request).await?.text().await?;
```

Comment:
Is this no longer possible with isahc? I don't mind changing tests to use reqwest, just curious.

Reply:
It is possible, however reqwest has this nice convenient function to set headers from http::header::HeaderMap directly, while with isahc I would have to reimplement it manually.